pkg/commands/standard/raw.go

```diff
@@ -0,0 +1,224 @@
+	command.PersistentFlags().StringToStringP("body-values", "", nil, "A set of key=value pairs that well be compiled into a json object. A dot (.) inside the key is used to traverse into nested objects. "+
+		"The key suffixes :bool or :int can be appended to the key to inject the value into the json structure as boolean or integer respectively.")
+	command.PersistentFlags().String("body", "", "The content of the body as a string")
+	command.PersistentFlags().String("body-file", "", "A file path pointing to a file containing the body of the http request")
+	command.PersistentFlags().String("output-file", "", "A file path pointing to where the response body shoule be written")
+	return command
+}
+
+//makeRawApiCall makes the actual call and writes the output to the supplied io.Writer
+func makeRawApiCall(restCaller *rest.RestCaller, method string, url string, queryParams map[string]string, body io.ReadCloser, mimeType string, out io.Writer) (*http.Response, error) {
+	defer body.Close()
+	//Make the request
+	fullUrl := restCaller.CreateUrl(url, queryParams)
+	req, err := http.NewRequest(method, fullUrl.String(), body)
+	req.Header.Add("Content-Type", mimeType)
+	res, err := restCaller.CallRawAndFollowRedirect(req)
+	if err != nil {
+		log.Fatal(err)
+	}
+	defer res.Body.Close()
+	io.Copy(out, res.Body)
+	return res, err
+}
+
+// getBodyFromFlags returns a ReadCloser that represents the body regardless of the parameters used
+func getBodyFromFlags(bodyFile string, bodyString string, bodyParams map[string]string) (body io.ReadCloser, mimeType string, error error) {
+	if multipleBodyParamsSet(bodyString, bodyFile, bodyParams) {
+		return nil, "", errors.New("Only one of body-path, body-string, and body-params can be used at one time")
+	}
+	if bodyString != "" {
+		mimeType := detectStringMimeType(bodyString)
+		return ioutil.NopCloser(strings.NewReader(bodyString)), mimeType, nil
+	}
+	if bodyFile != "" {
+		mimeType := detectFileMimeType(bodyFile)
+		file, err := os.Open(bodyFile)
+		return file, mimeType, err
+	}
+	if bodyParams != nil && len(bodyParams) > 0 {
+		body, err := buildBodyFromParams(bodyParams)
+		return body, "application/json", err
+	}
+	return ioutil.NopCloser(strings.NewReader("")), "", nil
+}
+
+func buildBodyFromParams(params map[string]string) (io.ReadCloser, error) {
+	rootJSonNode := make(map[string]interface{})
+	for key_, value := range params {
+		keyNameAndType := strings.Split(key_, ":")
+		key := keyNameAndType[0]
+		valueType := "string"
+		if len(keyNameAndType) > 1 {
+			valueType = keyNameAndType[1]
+		}
+		keyParts := strings.Split(key, ".")
+		jsonNode := rootJSonNode
+		for i := 0; i < len(keyParts)-1; i++ {
+			keyPart := keyParts[i]
+			if jsonNode[keyPart] == nil {
+				newNode := make(map[string]interface{})
+				jsonNode[keyPart] = newNode
+				jsonNode = newNode
+			}
+		}
+		lastKey := keyParts[len(keyParts)-1]
+		switch valueType {
```

Comment:
You can't send floats now right?
Maybe have number instead of int as it's called number in JSON.
Then you have to check if it can be parsed as an int or a float.